src/external-types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare const DATA_READY_HOOK: string;
```

Comment:
why is this file required?

Reply:
Good question 🤔 
I left this by accident